dask_geomodeling/raster/elemwise.py

```diff
@@ -730,9 +730,12 @@ def process(*args):
                 data_list.append(data["values"])
                 no_data_values.append(data["no_data_value"])
 
-        dtype = np.result_type(*data_list)
+        dtype = args[0]['dtype']
```

Comment:
Best to not include `args[0]` in the above for loop. Can you change the `process` call signature to
```
def process(process_kwargs, *args):
```